torch/_torch_docs.py

```diff
@@ -3471,10 +3473,26 @@ def merge_dicts(*dicts):
 :attr:`out` is of size :math:`(z \times n)`, where :math:`z` is the total number of
 non-zero elements in the :attr:`input` tensor.
 
+**When** :attr:`as_tuple` **is true:**
+
+Returns a tuple of 1-D tensors, one for each dimension in :attr:`input`,
+each containing the indices (in that dimension) of all non-zero elements of
+:attr:`input` .
+
+If :attr:`input` has `n` dimensions, then the resulting tuple contains `n` tensors
```

Comment:
shouldn't this explain the 0-dimensional exception case?

Reply:
Yes, good point